ksqldb-functional-tests/src/test/resources/historical_plans/suppress_-_should_drop_events_with_no_key/6.1.0_1595888718687/spec.json

```diff
@@ -0,0 +1,154 @@
+{
+  "version" : "6.1.0",
+  "timestamp" : 1595888718687,
+  "path" : "query-validation-tests/suppress.json",
+  "schemas" : {
+    "CTAS_OUTPUT_0.KsqlTopic.Source" : "STRUCT<COL1 VARCHAR> NOT NULL",
+    "CTAS_OUTPUT_0.Aggregate.GroupBy" : "STRUCT<ID VARCHAR, ROWTIME BIGINT> NOT NULL",
+    "CTAS_OUTPUT_0.Aggregate.Aggregate.Materialize" : "STRUCT<ID VARCHAR, ROWTIME BIGINT, KSQL_AGG_VARIABLE_0 BIGINT> NOT NULL",
+    "CTAS_OUTPUT_0.Suppress.Suppress" : "STRUCT<COUNT BIGINT> NOT NULL",
+    "CTAS_OUTPUT_0.OUTPUT" : "STRUCT<COUNT BIGINT> NOT NULL"
+  },
+  "testCase" : {
+    "name" : "should drop events with no key",
+    "inputs" : [ {
+      "topic" : "input_topic",
+      "key" : "k1",
+      "value" : {
+        "COL1" : "v1"
+      },
+      "timestamp" : 0
+    }, {
```

Comment:
I am not quite sure why these records are all on one line, this may just be the way that the historical plans are generated ? cc @agavra . But if you want to look at better formatted tests, they should be properly formatted in `ksqldb-functional-tests/src/test/resources/query-validation-tests/suppress.json`

Reply:
okay yeah I checked other existing historical tests and they all seemed to be formatted like this, but you can still just look at the `suppress.json` specified above for a more user-friendly reading experience :)

Reply:
Ah, I didn't realize the tests were also specified in that file. So much easier to read!

Reply:
I guess in retrospect, we should check in the generated files in exactly the format they're generated so that we get a clean diff in the future. Sorry for my confusion.